tools/schemapi/schemapi.py

```diff
@@ -1085,10 +1085,7 @@ def _default_wrapper_classes(cls) -> Iterator[type[SchemaBase]]:
 
     @classmethod
     def from_dict(
-        cls: type[TSchemaBase],
-        dct: dict[str, Any],
-        validate: bool = True,
-        _wrapper_classes: Iterable[type[SchemaBase]] | None = None,
+        cls: type[TSchemaBase], dct: dict[str, Any], validate: bool = True, **kwds: Any
```

Comment:
This to to align `Chart.from_dict`, without exposing the `_wrapper_classes` arg over there https://github.com/dangotbanned/altair/blob/419a4e944f231026322e2c4f137e7a3eb94735e8/altair/vegalite/v5/api.py#L2918

Reply:
See comment in corresponding `altair` file